lib/expansion_rules.rb

```diff
@@ -59,25 +59,25 @@ def recurring
     { document_type: :world_location,             fields: [:content_id, :title, :schema_name, :locale, :analytics_identifier] },
   ].freeze
 
-  def root_reverse_links
+  def reverse_links
     REVERSE_LINKS.values
   end
 
   def reverse_link_type(link_type)
     REVERSE_LINKS[link_type.to_sym]
   end
 
-  def un_reverse_link_type(link_type)
+  def unreverse_link_type(link_type)
```

Comment:
yeah it's a bummer.

So we have something like `child_taxons` which is a reverse link type, and when we look that up in the db we need to unreverse that name to be `parent_taxons` - so there's the context and why forward doesn't quite fit.

There is probably hope for a better name though, I'm just not sure how to make it succinct. 

eg could be `db_column_for_reverse_link_type` or `field_for_reverse_link_type` do you have any ideas too?